packages/cmr-client/tests/test-getUrl.js

```diff
@@ -2,26 +2,26 @@
 
 const test = require('ava');
 
-const getUrl = require('../getUrl');
+const { getSearchUrl } = require('../getUrl');
 
-test.serial('getUrl returns value according to cmrEnvironment param', (t) => {
-  t.is(getUrl('search', null, 'OPS'), 'https://cmr.earthdata.nasa.gov/search/');
-  t.is(getUrl('search', null, 'SIT'), 'https://cmr.sit.earthdata.nasa.gov/search/');
-  t.is(getUrl('search', null, 'UAT'), 'https://cmr.uat.earthdata.nasa.gov/search/');
+test.serial('getSearchUrl returns value according to cmrEnvironment param', (t) => {
+  t.is(getSearchUrl({ cmrEnv: 'OPS' }), 'https://cmr.earthdata.nasa.gov/search/');
+  t.is(getSearchUrl({ cmrEnv: 'SIT' }), 'https://cmr.sit.earthdata.nasa.gov/search/');
+  t.is(getSearchUrl({ cmrEnv: 'UAT' }), 'https://cmr.uat.earthdata.nasa.gov/search/');
 });
 
 test.serial('getUrl pulls cmrEnvironment from environment variables', (t) => {
   process.env.CMR_ENVIRONMENT = 'SIT';
-  t.is(getUrl('search'), 'https://cmr.sit.earthdata.nasa.gov/search/');
+  t.is(getSearchUrl(), 'https://cmr.sit.earthdata.nasa.gov/search/');
```

Comment:
Should update the test name here to `getSearchUrl() ..`

Reply:
8bc02f4f3